UnitsNet/UnitDefinitions/ThermalConductivity.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "ThermalConductivity",
+  "BaseUnit": "WattPerMeterKelvin",
+  "XmlDoc": "Thermal conductivity is the property of a material to conduct heat.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Thermal_Conductivity",
+  "Units": [
```

Comment:
Any prefix units to add?
Add these units?
http://www.unitconversion.org/unit_converter/thermal-conductivity.html